openapiart/openapiart.py

```diff
@@ -61,6 +61,7 @@ def __init__(
         self._get_info()
         self._get_license()
         self._document()
+        self._generate_proto_file()
```

Comment:
We should not generate proto by default (as some environments may not have such requirements).

I recommend exposing a func for generating protobuf separately.
```python
art.GenerateProtoDef(
    package_name="sanity"
)
```

Moreover, I would recommend performing `_generate_proto_file` durin both python and Go SDK (if it hasn't already been performed)